server/integration_tests/test_data/demo_fallback/query_4/chart_config.json

```diff
@@ -60,7 +60,7 @@
                       "Percent_Person_SleepLessThan7Hours",
                       "Percent_Person_Smoking"
                     ],
-                    "title": "Percentage of Population That Is Obese compared with other variables in United States",
+                    "title": "%  Population That Is Obese compared with other variables in United States",
```

Comment:
fix capitalization?

Reply:
In this case, the name of the SV ends before the "compared with ...". All of that portion seems Ok, right? Or are you suggesting to update the capitalization for: "% Population That Is Obese". I do notice two spaces between "%" and "Population" which I have now fixed/updated.